lib/emerald/parser.rb

```diff
@@ -35,13 +37,21 @@ def parse_whitespace(source)
     end
 
     def parse_atom(source)
-      pattern = /\A[a-z\-]+/
+      pattern = /\A[a-zA-Z\-]+/
       atom_value = pattern.match(source).to_s
       atom = Atom.new(atom_value)
       rest_of_source = drop(source, atom_value.size)
       [atom, rest_of_source]
     end
 
+    def parse_number(source)
+      pattern = /\d+/
+      number_value = pattern.match(source).to_s
```

Comment:
Ah ok, I was thinking that perhaps they would be dealt with as numbers at a later stage.  I changed it at line 50 :)